x-pack/plugins/security_solution/public/timelines/components/timeline/tabs/eql/index.tsx

```diff
@@ -161,127 +197,177 @@ export const EqlTabContentComponent: React.FC<Props> = ({
     [ACTION_BUTTON_COUNT]
   );
 
+  const header = useMemo(
+    () => (
+      <>
+        <EuiFlexItem grow={false}>
+          <StyledEuiFlyoutHeader
+            data-test-subj={`${activeTab}-tab-flyout-header`}
+            hasBorder={false}
+          >
+            <TabHeaderContainer data-test-subj="timelineHeader">
+              <EuiFlexGroup
+                className="euiScrollBar"
+                alignItems="flexStart"
+                gutterSize="s"
+                data-test-subj="timeline-date-picker-container"
+                responsive={false}
+              >
+                {timelineFullScreen && setTimelineFullScreen != null && (
+                  <ExitFullScreen
+                    fullScreen={timelineFullScreen}
+                    setFullScreen={setTimelineFullScreen}
+                  />
+                )}
+                <EuiFlexItem grow={false}>
+                  {activeTab === TimelineTabs.eql && (
+                    <Sourcerer scope={SourcererScopeName.timeline} />
+                  )}
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <SuperDatePicker
+                    width="auto"
+                    id={InputsModelId.timeline}
+                    timelineId={timelineId}
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <TimelineDatePickerLock />
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </TabHeaderContainer>
+          </StyledEuiFlyoutHeader>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EqlQueryBarTimeline timelineId={timelineId} />
+        </EuiFlexItem>
+      </>
+    ),
+    []
```

Comment:
The dependencies seem to be missing here

Reply:
Consider extracting this into its own component

Reply:
Apparently after my Mac updated, my vscode was quarantined and extensions stopped functioning properly. This helped me realize it.. thanks 😅

Reply:
done https://github.com/elastic/kibana/pull/180972/commits/3678fb1dcdbf3aaa0ceea82eb0e83d867164ef7c